std/zlib.d

```diff
@@ -315,7 +315,14 @@ class Compress
   public:
 
     /**
-     * Construct. level is the same as for D.zlib.compress(). header can be used to make a gzip compatible stream.
+     * Construct.
+     *
+     * Params:
+     *    level = the same as for $(D std.zlib.compress())
+     *    header = can be used to make a gzip compatible stream. Defaults to HeaderFormat.deflate.
```

Comment:
This description is not very clear. It should describe what the parameter *does*, not that it "can be used to" do something.